features/events/api/src/main/java/org/opennms/netmgt/events/api/Events.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2002-2014 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+/**
+ * @author ms043660 (Malatesh.Sudarshan@cerner.com)
+ */
+package org.opennms.netmgt.events.api;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.opennms.core.ipc.sink.api.Message;
+import org.opennms.netmgt.xml.event.Event;
+
+import com.google.common.primitives.Bytes;
+
+@XmlRootElement(name = "event")
+public class Events implements Message {
```

Comment:
Would it be possible to use existing `org.opennms.netmgt.xml.event.Log` and `org.opennms.netmgt.xml.event.Event` objects instead of defining new ones?

Reply:
Yes, org.opennms.netmgt.xml.event.Log is a very good option. Rather than converting logs at receiving end i would change these accordingly by updating it.Thanks for the info

Reply:
@j-white : Sorry jesse i got carried away.why new Events class was written because org.opennms.netmgt.xml.event.Log or org.opennms.netmgt.xml.event.Event doesn't implement org.opennms.core.ipc.sink.api.Message hence had to write new Events class in order to use MessageConsumer

Reply:
Can we just implement those interfaces on the existing objects?